### PR TITLE
fix bug when choose cluster due to no poolname

### PR DESCRIPTION
**Pull request by cxt90730**

### CODE REVIEW

**Comment by cannium** [comment]
Should also fix `rowKey`
https://github.com/journeymidnight/yig/blob/5df1f5f409d34686eec5b9d862da56bc8fbc15f6/meta/cluster.go#L10-L21

**Comment by cxt90730** [comment]
> Should also fix `rowKey`
> https://github.com/journeymidnight/yig/blob/5df1f5f409d34686eec5b9d862da56bc8fbc15f6/meta/cluster.go#L10-L21

I have committed in this pr...

File: meta/cluster.go [hunk 1]
```diff
@@ -7,11 +7,11 @@ import (
 	"github.com/journeymidnight/yig/redis"
 )
 
-func (m *Meta) GetClusters() (cluster []Cluster, err error) {
+func (m *Meta) GetClusters(pool string) (cluster []Cluster, err error) {
 	rowKey := "cephClusters"
 	getCluster := func() (c interface{}, err error) {
 		helper.Logger.Info("GetClusters CacheMiss")
-		return m.Client.GetClusters()
+		return m.Client.GetClusters(pool)
```
Comment (cxt90730) [inline meta/cluster.go]: @cannium

**Comment by cannium** [comment]
Close, prefer #216

**Pull request closed without merging**